build/jemalloc.m4

```diff
@@ -56,7 +56,7 @@ if test "$enable_jemalloc" != "no"; then
   if test "$jemalloc_base_dir" != "/usr"; then
     TS_ADDTO(CPPFLAGS, [-I${jemalloc_include}])
     TS_ADDTO(LDFLAGS, [-L${jemalloc_ldflags}])
```

Comment:
Should we remove this line then? Looks like a duplicate

Reply:
good catch!